fedimint-server/src/net/api.rs

```diff
@@ -266,5 +266,16 @@ fn server_endpoints() -> Vec<ApiEndpoint<FedimintConsensus>> {
                 Ok(fedimint.get_config_with_sig(dbtx).await)
             }
         },
+        api_endpoint! {
+            "/upgrade",
+            async |fedimint: &FedimintConsensus, _dbtx, _v: (), has_auth| -> () {
+                if has_auth {
+                    fedimint.signal_upgrade().await.map_err(|_| ApiError::server_error("Unable to send signal to server".to_string()))?;
```

Comment:
If we're able to send over this channel, we know that the signaling succeeded? We don't need to wait for any response from `fedimintd`?

Reply:
Yes it means the `ApiEvent` is in channel that triggers an epoch with the signal in the consensus thread.

Reply:
We use the same channel for users who submit txs.

Reply:
Cool. Thanks for explaining.